packages/components/src/styling/src/theming/createThemeVars.ts

```diff
@@ -103,12 +103,14 @@ function appendColorSchemes<C, L, D>(
 }
 
 function renderBucket(scope: string, bucket: VarsBucket) {
-    const element = document.createElement("style");
+    if(isBrowser) {
```

Comment:
missing a space between `if` and `(`

Reply:
Instead of scoping the whole code, you should exit at the beginning if it's on the server side.